pkgs/development/libraries/openal-soft/default.nix

```diff
@@ -1,7 +1,9 @@
 { lib, stdenv, fetchFromGitHub, cmake, pkg-config
 , alsaSupport ? !stdenv.isDarwin, alsa-lib
 , dbusSupport ? !stdenv.isDarwin, dbus
-, pipewireSupport ? !stdenv.isDarwin, pipewire
+# error: 'isnormal' was not declared in this scope; did you mean 'std::isnormal'?
+# https://github.com/NixOS/nixpkgs/issues/183774
+, pipewireSupport ? !stdenv.isDarwin && stdenv.hostPlatform == stdenv.buildPlatform, pipewire
```

Comment:
Meh, it's not great to disable it altogether, but I did the same in my own tree. I can live with it for now